web/setups/serversyncscript.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Copyright (c) 2017 University of Illinois, NCSA.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the
+ * University of Illinois/NCSA Open Source License
+ * which accompanies this distribution, and is available at
+ * http://opensource.ncsa.illinois.edu/license.html
+ */
+
+// Server side scripts to handle the client resquests
```

Comment:
This script works on the POST request with the proper JSON which consist of hostname and client-secret string provided which is being generated by clientsyncscript.php. 
If the client joined for the first time i.e client-secret is blank, this script returns a generated md5 string which is stored in the client side config.php file. this will stop from accidentally requesting another id for second time.